Tests/ApolloCodegenTests/GlobTests.swift

```diff
@@ -54,7 +60,8 @@ class GlobTests: XCTestCase {
     // when
     try create(files: [
       baseURL.appendingPathComponent("file.one").path,
-      baseURL.appendingPathComponent("file.two").path
+      baseURL.appendingPathComponent("file.two").path,
+      baseURL.appendingPathComponent("other/file.one").path
```

Comment:
complete wonkery: I like adding trailing commas to lists like this because then when you add or delete an item you don't have the comma change in the diff 🤓